Automattic/wp-calypso
# Add calypso.live comment to PRs

#### Background

We have a GitHub action that posts a link to the live branch in the form `calypso.live?branch=...`. 

When we were running e2e tests in TeamCity, that same link was used by the tests to load the live branch. However, since we moved to TeamCity there is also the form `calypso.live?build=...` which has a few benefits (see p4TIVU-9IM-p2)

#### Changes proposed in this Pull Request

* Add bash script to post the new calypso.live link from a TeamCity build
* Drop the GitHub action that was posting the link in the old form

#### Testing instructions

See TeamCity comments in this PR:

When the `Docker image` build starts it should show
![image](https://user-images.githubusercontent.com/975703/120802869-3786e480-c543-11eb-984d-5d5e784282b4.png)

Once the build is done, it should show:

![image](https://user-images.githubusercontent.com/975703/120970049-333d1000-c76b-11eb-9291-1599c0e84eac.png)
